gr-utils/python/utils/plot_data.py

```diff
@@ -28,6 +28,24 @@
 
 import numpy
 
+try:
```

Comment:
This needs `import argparse`

Reply:
yep, was also missing some other modules under pylab